nilearn/datasets/atlas.py

```diff
@@ -1840,6 +1843,30 @@ def fetch_atlas_surf_destrieux(
     )
 
 
+def load_sample_atlas_surf_destrieux():
```

Comment:
should probably provide `data_dir` to pass to `fetch_atlas_surf_destrieux` or ship sample data with nilearn